website/source/docs/providers/mysql/r/database.html.markdown

```diff
@@ -0,0 +1,54 @@
+---
+layout: "mysql"
+page_title: "MySQL: mysql_database"
+sidebar_current: "docs-mysql-resource-database"
+description: |-
+  Creates and manages a database on a MySQL server.
+---
+
+# mysql\_database
+
+The ``mysql_database`` resource creates and manages a database on a MySQL
+server.
+
+~> **Caution:** The ``consul_database`` resource can completely delete your
```

Comment:
Haha that's a pretty great typo. I can only imagine what was going on in my brain when I typed that. Thanks, will fix!